### DISCUSSION TITLE
Warehouses & Shipping Zones

### DISCUSSION THREAD

**Discussion by simon-online** [mirumee/saleor-dashboard#2103] [Q&A] [answered]
There's note in the Dashboard Manual docs that says:

**Note**
Currently, it's only possible to assign a warehouse to a single shipping zone. You can assign multiple warehouses to the same shipping zone.

Is this still the case? Because when I tested this out with v3.3 of saleor-dashboard and saleor I was able to assign the same warehouse to 2 different shipping zones. Is this properly supported in saleor?

Can anyone confirm if this is now properly supported?

**Comment by timuric** [comment, marked as answer]
Looks like we have outdated docs, thanks for reporting @simon-online I have updated the docs https://github.com/saleor/saleor-docs/pull/438
